flink-test-utils-parent/flink-test-utils-junit/src/main/java/org/apache/flink/core/testutils/FlinkAssertions.java

```diff
@@ -60,32 +60,13 @@ private FlinkAssertions() {}
                         .anySatisfy(
                                 cause ->
                                         assertThat(cause)
+                                                .as(
+                                                        "Any cause is instance of class '%s' and contains message '%s'",
+                                                        clazz, containsMessage)
```

Comment:
Why is this set here and not on `assertThatChainOfCauses`?
(And why is it necessary at all; is the assertj assertion message so bad on it's own?)

Reply:
It would still be better to set this at `assertThatChainOfCauses` because then the message is printed as part of the first line.
I have to say though that the assertj behavior here is pretty disappointing; it doesn't even document all assertions on it's own...